hoarder-linux.sh

```diff
@@ -254,6 +267,10 @@ function update {
     wget -q "https://github.com/hoarder-app/hoarder/archive/refs/tags/v${RELEASE}.zip"
     unzip -q v${RELEASE}.zip
     mv hoarder-${RELEASE} ${INSTALL_DIR}
+    # https://github.com/hoarder-app/hoarder/issues/967
+    if [[ $(corepack -v) < "0.31.0" ]]; then
```

Comment:
This will break when corepack reaches `0.100` but I think it's the problem of future me :D